rootdoc.txt

```diff
@@ -31,4 +31,9 @@ with asynchronous execution:
    implementations
  - [[monix.reactive.subjects]] exposes [[monix.reactive.subjects.Subject Subject]]
    and [[monix.reactive.subjects.ConcurrentSubject ConcurrentSubject]],
-   which are implementations of hot observables that function both as input and output
\ No newline at end of file
+   which are implementations of hot observables that function both as input and output
+
+'''[[monix.tail]]''' exposes [[monix.tail.Iterant Iterant]] for purely functional pull based streaming.
+
+You can control evaluation with type you choose - be it [[monix.eval.Task Task]], [[monix.eval.Coeval Coeval]], [[https://typelevel.org/cats-effect/api/cats/effect/IO.html cats.effect.IO]]
+or your own as long as you provide [[https://typelevel.org/cats-effect/typeclasses/sync.html cats.effect.Sync]] instance.
```

Comment:
`Iterant` isn't only about `Sync`, that's too specific. I think a simple mention of the `cats-effect` type classes should do.

Reply:
Oh, right, depending on operations you use it could be just `Applicative`